frontend/workflows/projectSelector/src/hello-world.tsx

```diff
@@ -190,25 +214,31 @@ const ProjectSelector = () => {
     setCustomProject("");
   };
 
+  const hasError = state.error !== undefined && state.error !== null;
+
   return (
     <DispatchContext.Provider value={dispatch}>
       <StateContext.Provider value={state}>
         <StyledSelectorContainer>
           <StyledWorkflowHeader>
+            {/* TODO: change icon to match design */}
             <LayersIcon />
             <StyledWorkflowTitle>Dash</StyledWorkflowTitle>
           </StyledWorkflowHeader>
           <StyledProgressContainer>
             {state.loading && <LinearProgress color="secondary" />}
           </StyledProgressContainer>
           <Divider />
-            <StyledProjectTextField
-              disabled={state.loading}
-              placeholder="Add a project"
-              value={customProject}
-              onChange={e => setCustomProject(e.target.value)}
-              onKeyDown={e => e.key === "Enter" && handleAdd()}
-            />
+          {/* TODO: add plus icon in the text field */}
+          <StyledProjectTextField
+            disabled={state.loading}
+            placeholder="Add a project"
+            value={customProject}
+            onChange={e => setCustomProject(e.target.value)}
+            onKeyDown={e => e.key === "Enter" && handleAdd()}
+            helperText={state.error?.message}
+            error={hasError}
```

Comment:
@dschaller, updated to use the TextField's component helperText and error fields.

Reply:
<img width="245" alt="Screen Shot 2021-06-24 at 4 31 01 PM" src="https://user-images.githubusercontent.com/39421794/123328465-9966b880-d509-11eb-91a0-4cb1bd22c8e1.png">